libs/ui/src/components/layouts/FormSegment/FormSegment.styles.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+export const StyledFormSegment = styled.section`
+  width: 100%;
+  margin-bottom: 2.4rem;
+  .segment-title {
```

Comment:
To keep consistent with spacing pattern in component lib
```suggestion

  .segment-title {
```